oak/common/policy.h

```diff
@@ -47,6 +47,10 @@ oak::policy::Label DeserializePolicy(const std::string& serialized_policy);
 // provided authorization bearer token.
 oak::policy::Label AuthorizationBearerTokenPolicy(const std::string& authorization_token);
 
+// Creates a policy that only allows declassifying data for modules that can present the
```

Comment:
I don't think modules "present" the attestation, rather the Oak Runtime knows the hash of the module making a request, and will allow a module with sha256 `123` to declassify tags `wasm_sha256: 123`. Also note that any node will be able to read the data, it's just the declassification that is gated by the module identity.

Reply:
Revised to indicate matching the attestation instead.